registry.json

```diff
@@ -3871,5 +3871,19 @@
     "telegram": "w_o_u_t",
     "operatorName": "Wout",
     "github": "tiligallery"
+  },
+  {
+    "ticker": "5HARE",
+    "poolId": "73a527af03d38b3659f20b263000a1f72d3f3b867df5a2bb0e5516f1",
+    "social": {
+      "website": "https://5hare-pool.5pecia1.dev",
+      "twitter": "",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "pdpxpd",
+    "operatorName": "5pecia1",
```

Comment:
Please change this to be either your real full name, real first name or leave empty